backend/workers/simulation_worker.py

```
@@ -0,0 +1,135 @@
+                await mark_job_failed(job_id, str(e), retry=True)
+            finally:
+                db.close()
+    
+    while True:
+        try:
+            job_data = await dequeue_job()
+            
+            if job_data:
+                job_id = job_data["job_id"]
+                logger.info(f"[SIMULATION_WORKER] Processing job: job_id={job_id}")
+                asyncio.create_task(process_with_semaphore(job_data))
```

Ruff (0.14.8)

Store a reference to the return value of `asyncio.create_task`

(RUF006)